img-buildkite-64.json

```diff
@@ -1,5 +1,10 @@
 [
   {
+    "name": "python-black",
+    "image": "img-buildkite-64/python-black.png",
+    "category": "Buildkite",
+    "aliases": []
+  },
     "name": "sidekiq",
```

Comment:
The JSON is missing an opening brace before here - looks a merge issue!

Reply:
I just quickly added that in @ticky!